signing/signer.go

```diff
@@ -24,6 +24,7 @@ const (
 	BALLOT   = 2
 	HARE     = 3
 	POET     = 4
+	MARRIAGE = 5
```

Comment:
Should this be its own domain? For me this belongs to `ATX`?

Reply:
I'm not sure. The ATX domain is used for signing ATXs. The MARRIAGE domain is used to sign the marriage certs. It's included in an ATX but it's a different thing. I think there's no harm in it being a separate domain, is there?